src/Spout/Writer/Style/Style.php

```diff
@@ -269,16 +269,25 @@ public function shouldWrapText()
     }
 
     /**
+     * @param boolean $shouldWrap Should the text be wrapped
      * @return Style
      */
-    public function setShouldWrapText()
+    public function setShouldWrapText($shouldWrap = true)
     {
-        $this->shouldWrapText = true;
+        $this->shouldWrapText = $shouldWrap;
         $this->hasSetWrapText = true;
         return $this;
     }
 
     /**
+     * @return boolean
```

Comment:
`@param bool` (for consistency, as we've been using "bool" instead of "boolean" everywhere and this is what PHP uses too)